src/app/forms/templates/general/weights.template.ts

```diff
@@ -0,0 +1,64 @@
+import { FormNode, FormNodeTypes } from '../../services/dynamic-form.types';
+
+export function generateWeights(prefix: boolean = false, vehicleType: string, prefixName?: string, ): FormNode[] {
```

Comment:
There are 3 checks for the prefix here:
1. In the method parameters `prefix: boolean = false, prefixName?: string`
2. In the individual lines this is used at `prefix ? addPrefix('gbWeight', prefixName) : 'gbWeight'`
3. In the helper method `if (prefix) {`

I think we could summarize all this in the helper method like so:

- Accept just one optional `prefix` parameter for `generateWeights()`
`export function generateWeights(vehicleType: string, prefix?: string): FormNode[] {`

- Remove the extra check on each line and just call the helper method
`name: addPrefix('gbWeight', prefix)`

- Abstract the check to the helper method, it's a helper method, right? It should help!
```TypeScript
function addPrefix(string: string, prefix?: string): string {
  return prefix
    ? prefix + string.charAt(0).toUpperCase() + string.slice(1);
    : string;
} 
```

Reply:
Perfect, I'll make the change